login-workflow/src/components/password/ChangePasswordModal.tsx

```diff
@@ -44,7 +45,7 @@ export const ChangePasswordModal: React.FC = () => {
     const securityHelper = useSecurityActions();
     const theme = useTheme();
     const sharedClasses = useDialogStyles();
-    const matchesXS = useMediaQuery(theme.breakpoints.down('xs'));
+    const matchesXS = useMediaQuery(theme.breakpoints.down('sm'));
```

Comment:
If we're switching all of these to `sm` breakpoints instead of `xs` we should probably rename the variable to reflect that. Why was it necessary to switch the breakpoint?  Are there new breakpoint definitions in mui5 that break things here?

Reply:
Breakpoints range has changed in MUIv5. Let me update variable name.

<img width="852" alt="Screenshot 2022-03-10 at 12 55 19 PM" src="https://user-images.githubusercontent.com/10433274/157610103-f960c46b-69ac-4d64-8298-d7ea04943dcc.png">